package.json

```diff
@@ -3,14 +3,21 @@
     "displayName": "Kubernetes",
     "description": "Develop, deploy and debug Kubernetes applications",
     "version": "0.1.0",
-    "publisher": "Microsoft",
+    "preview": true,
+    "publisher": "ms-kubernetes-tools",
     "engines": {
         "vscode": "^1.13.0"
     },
     "license": "MIT",
     "categories": [
+        "Azure",
```

Comment:
I'm not clear what `categories` means here. If it means, "the publisher", ok. But these commands are not specific to Azure, are they? they work anywhere we can pick up your config file, correct? 

If that's right, then we need this to say something different, like `Kubernetes`. But I am ignorant what this section is supposed to mean.